src/app/space/settings/areas/areas.component.ts

```diff
@@ -14,17 +15,22 @@ import { IModalHost } from '../../wizard/models/modal-host';
   styleUrls: ['./areas.component.less']
 })
 export class AreasComponent implements OnInit, OnDestroy {
+
+  @ViewChild('createAreaDialog') createAreaDialog: CreateAreaDialogComponent;
+  @ViewChild(ModalDirective) modal: ModalDirective;
+
+  private bsModalRef: BsModalRef;
   private context: Context;
   private areas: Area[];
   private emptyStateConfig: EmptyStateConfig;
   private listConfig: ListConfig;
   private areaSubscription: Subscription;
   private selectedAreaId: string;
-  @ViewChild('createArea') createArea: IModalHost;
 
   constructor(
     private contexts: ContextService,
-    private areaService: AreaService) {
+    private areaService: AreaService,
+    private modalService: BsModalService) {
```

Comment:
Is this used? Does it need to be injected?

Reply:
Yes ! It gets used a few methods below in createArea()

Reply:
Where is this createArea function? I see this parameter/field added to the constructor, but I don't see any added hunks in this commit that reference it. Ctrl+F for modalService doesn't turn up any other usages.

Reply:
Ah, sorry again, I was looking in the wrong file. The service is referenced here below https://github.com/fabric8-ui/fabric8-ui/pull/2477/files#diff-60e5b75cacb1ac69f3a6573acba90c3dR48

Reply:
My bad, I misunderstood, I thought you were referencing AreaService. It's been dropped.